app/controllers/account/entries_controller.rb

```diff
@@ -66,6 +30,10 @@ def destroy
 
   private
 
+    def entryable_view_path(action)
+      @entry.entryable_type.underscore.pluralize + "/" + action.to_s
+    end
```

Comment:
For a mixed list of entries, `entries = [@trade, @transaction]`, we want to be able to do this in our views:

```erb
<% entries.each do |entry| %>
  <%= link_to account_entry_path(entry.account, entry) %>
<% end %>
```

This delegation helps us avoid:

```
<% entries.each do |entry| %>
  <% if entry.account_transaction %>
    <%= link_to account_transaction_path(entry.account, entry) %>
  <% elsif entry.account_trade? %>
   ...
<% end %>
```